search/lib/util/app.js

```diff
@@ -19,14 +19,18 @@ function extractTypeFromHref (href) {
   return type;
 }
 
-const createLink = (rel, href = '', title, type = 'application/json') => {
+const createLink = (rel, href = '', title, type = 'application/json', method = null) => {
   type = extractTypeFromHref(href) || type;
 
   if (href.includes('.native')) {
     href = href.replace('.native', '.xml');
   }
 
-  return { rel: rel, href: href, title: title, type: type };
+  const link = { rel: rel, href: href, title: title, type: type };
+  if (method) {
```

Comment:
What happens here if method is null?  That isn't an error condition, just something that can sometimes happen?

Reply:
If method isn't passed into the function that it's not added to the link at all.